python/pyproject.toml

```diff
@@ -222,5 +222,9 @@ ignore_missing_imports = true
 module = "pyparsing.*"
 ignore_missing_imports = true
 
+[[tool.mypy.overrides]]
+module = "numpy.*"
+ignore_missing_imports = true
```

Comment:
Why is numpy used directly?

Reply:
PyArrow ships with numpy. We use it to create empty arrays, but I think we can remove that!

Reply:
Removed!